foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/generator/DataRecordsModuleStep.java

```diff
@@ -110,6 +133,13 @@ public void the_profile_is_generated(String profileName) throws Exception {
         }, new Slf4jDelegate(logger), projectDir.toFile());
     }
 
+    @When("the pipelines are validated")
+    public void the_pipelines_are_validated() throws Exception {
```

Comment:
Q/S: Why did we add this? I don't see it in the scenarios.

Reply:
It was moved from lower down in the file. used to be on line 325.